src/api/subscriptions.js

```diff
@@ -257,6 +268,7 @@ export const SubscriptionFlows = {
   LINK_ACCOUNT: 'linkAccount',
   SHOW_LOGIN_PROMPT: 'showLoginPrompt',
   SHOW_LOGIN_NOTIFICATION: 'showLoginNotification',
+  SMART_SUBSCRIPTION_BUTTON: 'showSmartSubscriptionButton',
```

Comment:
I'm not convinced we need this.

Reply:
This was left over from previous implementation. Was not being used anywhere, removed.